ProjectSidewalk/SidewalkWebpage
# #2680 add uic logo

Resolves #2680
- Added png image of UIC logo (higher res, smaller res looked very pixelated) to landing page.
- Changed the padding css for the container-image from the right and left to be 5% of total screen size to make it more dynamic.

To test changes:
- See if it looks fine, and try on different screen sizes.



This is a brief description of the problem solved or feature implemented and how you implemented it.

##### Before/After screenshots (if applicable)


Before:
![Capture](https://user-images.githubusercontent.com/41341006/142559957-bdab9bbf-4b80-4711-8389-80cc7ad6afb5.PNG)
After:
![Capture1](https://user-images.githubusercontent.com/41341006/142559991-26571a18-3371-4376-a70d-e20c09773bfa.PNG)


##### Testing instructions
1. See if it looks fine, and try on different screen sizes.

##### Things to check before submitting the PR
